package.json

```diff
@@ -76,7 +76,7 @@
     "glob": "^7.0.5",
     "hoist-non-react-statics": "^1.2.0",
     "intl": "^1.2.5",
-    "lodash": "^4.17.1",
+    "lodash.escape": "^4.0.1",
```

Comment:
where and how is it used ? 
is it possible to make your own replace, it sounds pretty simple: https://github.com/lodash/lodash/blob/4.17.2/lodash.js#L14212